### DISCUSSION TITLE
Save and Export

### DISCUSSION THREAD

**Discussion by chenlung** [URUWorks/TeroSubtitler#88] [General]
Could you explain, @URUWorks, the difference between Export and Save in the menu?

**Comment by chenlung** [comment]
ACTitle.act appears to be the only format that will save from Tero Subtitler but not open.

**Reply by URUWorks** [reply]
Do you mean AQTitle? I'm going to review it.

**Reply by chenlung** [reply]
> Do you mean AQTitle? I'm going to review it.

Yes, sorry! 😊

**Comment by URUWorks** [comment]
> Could you explain, @URUWorks, the difference between Export and Save in the menu?

Of course, "Save" saves the entire subtitle file, "export" only saves the selected text or marked subtitles.

**Reply by chenlung** [reply]
> > Could you explain, @URUWorks, the difference between Export and Save in the menu?
> 
> Of course, "Save" saves the entire subtitle file, "export" only saves the selected text or marked subtitles.

Thanks! I remembered you explained something weeks ago elsewhere, but I forgot. 😆
